src/components/activePanel/activePanel.template.html

```diff
@@ -1,21 +1,35 @@
-<div class="active-panel">
+<div class="active-panel ng-hide" ng-show="d.currStatus !== d.states.NONE">
     <div class="flex-container">
         <div class="callee-status flex-col">
-            <div class="online-indicator"></div>
+            <div class="online-indicator">
+                 <!--ng-class="{-->
+                    <!--'offline': vm.student.presence==vm.userStatus.OFFLINE,-->
+                    <!--'online': vm.student.presence==vm.userStatus.ONLINE,-->
+                    <!--'idle': vm.student.presence==vm.userStatus.IDLE-->
+                 <!--}">-->
+            </div>
         </div>
-        <div class="callee-name flex-col" title="{}">
-            {{calleeName}}
+        <div class="callee-name flex-col">
+            {{d.calleeName}}
             <div class="call-duration">&nbsp;</div>
         </div>
         <div class="call-controls flex-col">
-            <!--<svg-icon name="call-mute-icon"></svg-icon>-->
-            <!--<div class="share-my-screen"></div>-->
-            <svg-icon ng-hide="true" name="active-panel-share-screen-icon"></svg-icon>
-            <ng-switch ng-hide="true" on="iama" class="show-other-screen">
-                <svg-icon ng-switch-when="teacher" name="active-panel-track-student-icon"></svg-icon>
-                <svg-icon ng-switch-when="student" name="active-panel-track-teacher-icon"></svg-icon>
-            </ng-switch>
-            <call-btn ng-model="callBtnModel"></call-btn>
+            <div ng-click="d.viewOtherUserScreen()"
+                 class="show-other-screen"
+                 disable-click-drv="d.shareScreenBtnsEnable"
+                 ng-class="{disabled: !d.shareScreenBtnsEnable}">
+                <svg-icon ng-if="d.isTeacher" name="active-panel-track-student-icon"></svg-icon>
+                <svg-icon ng-if="!d.isTeacher" name="active-panel-track-teacher-icon"></svg-icon>
```

Comment:
ng-if="d.isTeacher"
ng-if="!d.isTeacher"
I prefer ng-switch to prevent flickers in UI